drivers/bluetooth/hci/Kconfig.nxp

```diff
@@ -25,6 +25,21 @@ config HCI_NXP_SET_CAL_DATA_ANNEX100
 
 if BT_H4_NXP_CTLR
 
+config FLASH
```

Comment:
I don't think those config should be set under `BT_H4_NXP_CTLR`.
`BT_H4_NXP_CTLR` is not selected on RW, `BT_NXP` is.
`BT_H4_NXP_CTLR` is for host platform that will communicate with a tranceiver through the uart h4 driver.

Lyle from Tony's team should review this too, @dleach02 do you know Lyle's tag on github ?

Reply:
Yes I agree